lib/src/bindings.dart

```diff
@@ -31,3 +31,15 @@ abstract class JsEvent {
   external String get resource;
   external dynamic get timestamp;
 }
+
+//TODO: Perhaps should be exposed in pkg/node_interop?
```

Comment:
Sure – although in this case, it seems easy to just expose the interop object `Console` directly, so I avoided that.